parsigs/parse_sig_api.py

```diff
@@ -6,9 +6,8 @@
 import re
 import logging
```

Comment:
@sigmagamma looks great! 
one small thing - add the new `krovetzstemmer` to the `dependencies` in the `pyproject.toml` file (this is for building the dist that is sent to Pypi)